utils/tests/artifactoryconsts.go

```diff
@@ -100,6 +100,7 @@ const (
 var Repo1 = "jfrog-cli-tests-repo1"
 var Repo2 = "jfrog-cli-tests-repo2"
 var Repo1And2 = "jfrog-cli-tests-repo*"
+var Repo1And2Placeholder = "jfrog-cli-tests-repo(*)"
```

Comment:
Let's add the following comment for this new variable:
// This is not an actual repository. This is a pattern, which is meant to be used in both Repo1 and Repo2.